.github/workflows/cancel-previous-runs.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto Cancel Previous CI Runs
+on:
+  create:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - latest
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@master
```

Comment:
Just had a quick look to see what this plugin did and all looks good security wise 👍 Was thinking there was a possibility it could leak the github_token accidently/deliberately.